yours-org/yours-wallet
# feat(provider): add signTransaction method - issue #19

@shruggr Steps to test:

1. Pull down `feat/sign-transactions-method`
2. `npm i`
3. `npm run build`
4. load build folder into unpacked chrome://extensions
5. Navigate to random website other than google (they seem to block injections).
6. Connect the provider with await `window.panda.connect()`
7. Call `window.panda.signTransaction()` and pass in params.

Example:
```
await panda.signTransaction({
rawtx: "010000000146f187d4cdb074d8f2c0c889699aaf2612a03bb1b4c28e2bae27a4a1689c4e2a0100000000ffffffff02f4010000000000001976a9143da10df305df0db0f1f5621c4026a7ce5867bfaf88acba6f0000000000001976a914e21750ff1e380e0d3e84eedeaa09b47531997ba988ac00000000",
  vin: 0,
  sigHashTypeNumber: 1,
  keyType: "ord",
  outputScript: "76a914e21750ff1e380e0d3e84eedeaa09b47531997ba988ac",
  outputSats: 500})
```

This should invoke a popup that you can approve and then ultimately it should log the signature.
